src/Framework/Framework/Resources/Scripts/api/api.ts

```diff
@@ -98,10 +100,13 @@ export function invoke<T>(
     }
 
     refreshArgs()
-    ko.computed(() =>
-        refreshTriggers(args).map(trigger => typeof trigger == "string" ? eventHub.get(trigger)() : trigger())
-    )
+    const subscription = ko.computed(
+            () => refreshTriggers(args).map(trigger => typeof trigger == "string" ? eventHub.get(trigger)() : trigger())
+        )
```

Comment:
You can use  https://github.com/riganti/knockout/blob/master/src/binding/defaultBindings/ifIfnotWith.js#L67
```suggestion
        , null, { disposeWhenNodeIsRemoved: lifetimeElement })
```